src/windows/installer.nsi

```diff
@@ -185,7 +185,7 @@ FunctionEnd
 Function check_instance
 
   check:
-  FindProcDLL::FindProc "qbittorrent.exe"
+  FindProcDLL::FindProc "$INSTDIR\qbittorrent.exe"
```

Comment:
So, if the PR is accepted, the NSIS plugin should be replaced, but these lines shouldn't be modified.